WordPress/Classes/System/WordPressAppDelegate.swift

```diff
@@ -401,8 +401,7 @@ extension WordPressAppDelegate {
 
     @objc func configureAppCenterSDK() {
         #if APPCENTER_ENABLED
-        MSAppCenter.start(ApiCredentials.appCenterAppId(), withServices: [MSDistribute.self])
-        MSDistribute.setEnabled(true)
+        AppCenter.start(withAppSecret: ApiCredentials.appCenterAppId(), services: [Distribute.self])
```

Comment:
This looks good to me.

I guess the only way to actually test it, though, is to ship another build to App Center and see if the this one shows the update prompt and if the process succeeds?

Reply:
Yeah, exactly – I'm sure we'll have one this build 😅